novawallet/Common/Services/TransactionSubscription/ExtrinsicProcessor+Matching.swift

```diff
@@ -90,6 +90,11 @@ extension ExtrinsicProcessor {
                 runtimeJsonContext: context
             )
 
+            let feeAsset = localAsset(
```

Comment:
Here we need to have orml specific parsing. More precisely the parser must match the chain and apply Hydra specific parsing.